### ISSUE TITLE
Hidden Tab Redirect not working

### ISSUE THREAD

**Issue by sinclairfr** [maykar/custom-header#60]
Example : 
```
    - conditions:
        user: test_user
      config:
        hide_tabs: '0,1,2'
        hide_config: true
        hide_help: true
        hide_unused: true
        hide_raw: true
        menu_hide: true
        voice_hide: true
        options_hide: true
        reverse_button_direction: true
        voice_dropdown: true
        menu_dropdown: true
        default_tab: '7'
        compact_mode: true
        disable_sidebar: true
        show_tabs: '7'
        locale: []
```
If the user goes to ...homeassistant.local/lovelace/0 our /lovelace/1 etc they can view the content. No redirection to the default tab 7.

**Comment by sinclairfr** [comment]
The version of Home Assistant 0.103.3 and Custom Header Installed: 1.1.2

**Comment by maykar** [comment]
Just released 1.1.3 to fix this issue.

I would also suggest using the config below instead of what you're doing. I realize that this might just be you testing out config, but just in case.

hide_tabs shouldn't be used with show_tabs, if you're hiding the buttons there's no reason to put them into the overflow/options menu (which you're also hiding), since all buttons are hidden you don't need to reverse their direction, and if all that is hidden as well as the sidebar why not just use kiosk mode.

Try this instead:
```
    - conditions:
        user: test_user
      config:
        kiosk_mode: true
        default_tab: '7'
        show_tabs: '7'
```

**Comment by sinclairfr** [comment]
You rock! 
Yes that was just me trying to find out what might be the cause of the issue so I ended up with a overly complicated config.
I wish I could find a way to help you out with your project but my javascript is rusty. I'll stick to bug reporting for now :)
Thanks again
